src/main/kotlin/blackjack/ui/UI.kt

```diff
@@ -1,21 +1,56 @@
 package blackjack.ui
 
+import blackjack.domain.DealerRule
 import blackjack.domain.Player
 import blackjack.domain.Players
+import blackjack.domain.WinningResult
 
 object UI {
 
-    fun drawFirstTurnMessage(players: Players) {
+    fun drawFirstTurnMessage(dealer: Player, players: Players) {
         val playerNames = players.list.joinToString(",") { it.name }
-        println("${playerNames}에게 각자 2장을 나누었습니다.")
+        println("${dealer.name}와 ${playerNames}에게 각자 2장을 나누었습니다.")
     }
 
     fun drawCardList(player: Player) {
-        println("${player.name}카드: ${player.currentCards().joinToString(", ") { it.toString() }}")
+        val playerCards = when (player.rule) {
+            is DealerRule -> player.currentCards().first().toString()
+            else -> player.currentCards().joinToString(", ") { it.toString() }
+        }
+        println("${player.name}카드: $playerCards")
+    }
+
+    fun drawDealerDrawMessage(dealer: Player) {
+        println("${dealer.name}는 16이하라 한장의 카드를 더 받았습니다.")
     }
 
     fun drawResult(player: Player) {
-        println("${player.name}카드: ${player.currentCards().joinToString(", ") { it.toString() }} - ${player.score}")
+        val playerCards = player.currentCards().joinToString(", ") { it.toString() }
+        println("${player.name}카드: $playerCards - ${player.score.value}")
+    }
+
+    fun drawRecordTitle() {
+        println("## 최종 승패")
+    }
+
+    fun drawRecord(result: WinningResult) {
+        val recordString = when (result.player.rule) {
+            is DealerRule -> {
+                buildString {
+                    if (result.winCount > 0) append("${result.winCount}승")
+                    if (result.loseCount > 0) append("${result.loseCount}패")
+                    if (result.drawCount > 0) append("${result.drawCount}무")
+                }
+            }
+            else -> when {
```

Comment:
```suggestion
            PlayerRule -> when {
```
- sealed interface/class는 꼭 타입으로 비교하지 않아도 됩니다. 
- when + sealed interface/class는 else 없이도 사용가능해요. 지금은 클래스를 명시해주는 편이 나중에 Rule이 추가될 경우에 좀 더 안전하지 않을까요?

[kotlin 의 sealed + when 문서](https://kotlinlang.org/docs/sealed-classes.html#sealed-classes-and-when-expression)를 가볍게 읽고 넘어가도 좋을 것 같습니다!